app/src/main/java/com/quarkworks/apartmentgroceries/profile/ProfileActivity.java

```diff
@@ -0,0 +1,126 @@
+package com.quarkworks.apartmentgroceries.profile;
+
+import android.content.Context;
+import android.content.Intent;
+import android.support.design.widget.CollapsingToolbarLayout;
+import android.support.v7.app.AppCompatActivity;
+import android.os.Bundle;
+import android.support.v7.widget.Toolbar;
+import android.view.Menu;
+import android.view.MenuItem;
+import android.view.View;
+import android.view.ViewGroup;
+import android.widget.ImageView;
+import android.widget.ListAdapter;
+import android.widget.ListView;
+
+import com.bumptech.glide.Glide;
+import com.quarkworks.apartmentgroceries.R;
+import com.quarkworks.apartmentgroceries.grocery.GroceryCell;
+import com.quarkworks.apartmentgroceries.service.DataStore;
+import com.quarkworks.apartmentgroceries.service.models.RGroceryItem;
+import com.quarkworks.apartmentgroceries.service.models.RUser;
+
+import io.realm.RealmBaseAdapter;
+import io.realm.RealmResults;
+
+public class ProfileActivity extends AppCompatActivity {
+    private static final String TAG = ProfileDetailActivity.class.getSimpleName();
```

Comment:
I think I'm going to review it again after I pull the request next time and make a checklist. 😁